app/javascript/elm/src/FixedSessionFilters.elm

```diff
@@ -62,6 +69,12 @@ type Msg
 update : Msg -> Model -> ( Model, Cmd Msg )
 update msg model =
     case msg of
+        UpdateLocationInput newValue ->
+            ( { model | location = newValue }, Cmd.none )
```

Comment:
This is picky so feel free to totally disregard. When updating values in a record I'd use `newKEY`. In this case `newLocation`